cardano-testnet/src/Parsers/Help.hs

```diff
@@ -0,0 +1,51 @@
+module Parsers.Help
+  ( HelpOptions(..)
+  , cmdHelp
+  , runHelpOptions
+  ) where
+
+import           Control.Monad (forM_)
+import qualified Data.List as List
+import           Options.Applicative
+import           Options.Applicative.Help.Core
+import           Options.Applicative.Help.Types (renderHelp)
+import           Options.Applicative.Types (OptReader (..), Option (..), Parser (..))
+import qualified System.IO as IO
+
+import           Cardano.CLI.Common.Parsers
+
+data HelpOptions = HelpOptions
```

Comment:
I'm highly suspect of this type. Can it be removed?

Reply:
The type is effectively `()` but is a distinct type for type-safety.  We already do a similar thing for `VersionOptions`.